anitya/db/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""
```

Comment:
Might be good to put a Copyright header in this and other new files. Unless the project has a global copyright claim?